src/data/constants/api.js

```diff
@@ -0,0 +1,14 @@
+import { configuration } from 'config';
+
+export const baseUrl = `${configuration.LMS_BASE_URL}/api`;
+
+/**
+ * bulkGradesUrlByCourseAndRow(courseId, rowId)
+ * returns the bulkGrades url with the given courseId and rowId.
+ * @param {string} courseId - course identifier
+ * @param {string} rowId - row/error identifier
+ * @return {string} - bulk grades fetch url
+ */
+export const bulkGradesUrlByCourseAndRow = (courseId, rowId) => (
+  `${baseUrl}/bulkGrades/course/${courseId}/?error_id=${rowId}`
```

Comment:
Not to be pedantic, but this isn't a constant. Should it go somewhere else?

Reply:
honestly, it should probably go into the LmsApiService file, and may well do so, once I clean that up.  I just really didn't want to go in and muck with that until I was ready to do more thorough overhauls in that file.